values.yaml

```diff
@@ -174,6 +174,9 @@ injector:
     annotations: {}
 
 server:
+  # if set to false, chart "mode" will define as "external", see _helpers.tpl
```

Comment:
```suggestion
  # If set to false, Vault server will not be installed. See vault.mode in
  # _helpers.tpl for implementation details
```

Just a small nit to make this documentation more relevant to the average consumer.